src/utils/editor-environment.js

```diff
@@ -23,6 +28,7 @@ export async function setUpEditorEnvironment() {
 		setUpGlobalErrorHandlers();
 		setBodyClasses();
 		await awaitGBKitGlobal();
+		setLogLevelFromGBKit();
```

Comment:
Hoisted from `src/utils/editor.jsx` to ensure messages from editor initialization log.